src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -128,6 +128,7 @@ class MLParams(
     val k: Option[Int] = None,
     val link: Option[String] = None,
     val maxIter: Option[Int] = None,
+    val numBuckets: Option[Int] = Some(2),
```

Comment:
I'd recommend reusing bucketizerNumBuckets  (or removing bucketizerNumBuckets in favor of numBuckets)

Reply:
I'd also recommend not specifying defaults since it'd be better to force ourselves to think about what we're setting these to in perf tests.